src/utils/artifact-generation.ts

```diff
@@ -22,6 +23,41 @@ export async function generateArtifacts(startScript: string): Promise<void> {
     throw result.error
   }
 
+  if (process.env.NEXUS_TYPEGEN_NEXUS_SCHEMA_IMPORT_PATH) {
```

Comment:
Big fat ugly but self-contained "feature".

Documented in README in system tests section